### DISCUSSION TITLE
How to display a toggle button?

### DISCUSSION THREAD

**Discussion by Demian101** [google/mdbook-i18n-helpers#98] [Q&A]
I want to know how to display a toggle button on my homepage after configuring the language folder, so that I can switch languages?

I looked at [dojo's mdbook](https://github.com/dojoengine/book/blob/main/theme/book.js), and they seemed to have manually written some JavaScript dom code?

Is there some convenient way for me to add this toggle button to mdbook?

PLS ~

**Comment by mgeisler** [comment]
It looks like this is a duplicate of #99.
